tests/prefilter-multibuf-multipkts/test.yaml

```diff
@@ -0,0 +1,17 @@
+requires:
+  min-version: 8
+
+args:
+- -k none
+
+checks:
+- filter:
```

Comment:
Should we include a check that `sid: 1` should not match?

Reply:
sid 1 was a leftover from #2093 where I found this bug